pkg/parser/parser.peg

```diff
@@ -363,6 +363,81 @@ StandaloneAttributes <- attributes:(ElementAttribute)+ BlankLine* EOF { // stand
     return types.NewAttributes(attributes)
 }
 
+// ------------------------------------------
+// Quoted Strings (between curly single or double quotes)
+// ------------------------------------------
+
+QuotedString <- qs:(SingleQuotedString / DoubleQuotedString) {
+    return qs, nil
+}
+
+SingleQuotedString <- "'`" elements:SingleQuotedStringElements "`'" {
+    return types.QuotedString{ Kind: types.SingleQuote, Elements: elements.([]interface{})}, nil
```

Comment:
let's create a `types.NewQuotedString(type, elements)` func in `pkg/types/types.go` to be consistent with the rest of the code in this file

Reply:
Done.